lib/mesh.js

```diff
@@ -2501,7 +2503,7 @@ Mesh.prototype._loadComponentSuites = function(config, callback) {
               if (suite[0]) delete suite[0].component.name;
               else delete suite.component.name;
               error = _this.__supplementSuites(packaged, suite, name, null);
-              return cb();
+              cb();
```

Comment:
we can leave this here, this is. ancient unused. functionality that should be removed